### PR TITLE
Localize fx3738 Add #9956 [v99] String freeze

### PR DESCRIPTION
**Pull request by lmarceau**
- Update text wallpaper feature ([Jira ticket FXIOS-3722](https://mozilla-hub.atlassian.net/browse/FXIOS-3722)) #9956
Simulator Screen Shot - iPhone 11 - 2022-03-01 at 11 53 14

- String freeze for https://github.com/mozilla-l10n/firefoxios-l10n/pull/129

### CODE REVIEW

**Review by electricRGB — APPROVED** [review]

**Comment by lmarceau** [comment]
I'll wait for the l10n PR to be merged before merging this PR here in case changes are needed.

**Pull request merged**
